src/ragger/conftest/base_conftest.py

```diff
@@ -212,8 +213,8 @@ def navigator(backend, firmware, golden_run):
         return NanoNavigator(backend, firmware, golden_run)
     else:
         # `firmware` fixture is generated by `pytest_generate_tests`, which controls it against the
-        # `FIRMWARES` list. So by design, if the firmware is not a Nano, it is a Stax
-        return StaxNavigator(backend, firmware, golden_run)
+        # `FIRMWARES` list. So by design, if the firmware is not a Nano, it is either Stax or Flex
+        return NBGLNavigator(backend, firmware, golden_run)
```

Comment:
I'm not sure NBGL is the right name for that.
Is it linked to the screen side and touch capability or to the UX lib used?
How will it be impacted once the LNX and LNSP start using NBGL?

Reply:
It is priamerly linked with the screen capabilities, although the code is also structured to mimic `NBGL` layers (`layouts` & `use_cases`).
I guess we _could_ integrate LNX/LNSP in it, but I'm not sure it would bring any benefits - except if we develop an additional layer overclassing the `use_cases`, with successive screens in a pre-defined order.

Reply:
Maybe `Touch` would be a better naming? `TouchNavigator`, `touch` module, ...